app/ParserTests/ParserTests.hs

```diff
@@ -0,0 +1,117 @@
+{-|
+Description: Test Course Requirement Parsers using HUnit Testing Framework.
```

Comment:
@PkHutch please advise about the right field to add to this docstring here ("module" or something).